versioned/version0_3_1.ts

```diff
@@ -389,4 +389,4 @@ class Version0_3_0 extends Versioned {
   }
 }
 
-export default Version0_3_0;
+export default Version0_3_1;
```

Comment:
Error:

example: `/KT1AxG32ZKBDazFXv2PxiL2HCJJm7MRs8cxg/history`

```
Unhandled Runtime Error
InvalidHexStringError: Invalid hex string "[object Object]" : Expecting even number of characters.

Source
versioned/version0_3_1.ts (316:33) @ content

  314 | status: "Non-executed;",
  315 | meta: content.execute_lambda.metadata
> 316 |   ? bytes2Char(content.execute_lambda.metadata)
      |               ^
  317 |   : "No meta supplied",
  318 | 
  319 | lambda: emitMicheline(
```
additional "Some" in the "content.execute_lambda.metadata"
for the version `< 0.3.1`, the "Some" seems to be handled by tzkt api.